docs/detections/visual-event-analyzer.asciidoc

```diff
@@ -24,12 +24,12 @@ To access events that can be visually analyzed:
 [role="screenshot"]
 image::images/kql-agent-type.png[]
 
-3. For the event that you want to analyze, click the **More actions** button and select **Analyze event**. The visual analyzer view appears.
+3. For the event that you want to analyze, click **Analyze event**. The visual analyzer view appears.
 +
 [role="screenshot"]
 image::images/analyze-event.gif[Shows analyze event option]
```

Comment:
Kind of a nitpick, but I don't think an animated GIF is necessary anymore, since the option isn't as complex/hidden as it used to be. The icon is visible now and all the user needs to do is click it. Maybe just a static image with a box or arrow to highlight the icon?

Reply:
Ah yes, good point @joepeeples. If the gif's too distracting, I have no problem replacing it. A static image with those annotations, or even an image containing the tooltip that appears when users hover over the analyze event icon, would work.

Reply:
Either of those options sound good (assuming the second option is also a static image showing the tooltip).